code/parachain/frame/ibc/ping/src/lib.rs

```diff
@@ -263,7 +252,7 @@ impl<T: Config + Send + Sync> Module for IbcModule<T> {
 		_relayer: &Signer,
 	) -> OnRecvPacketAck {
 		let success = "ping-success".as_bytes().to_vec();
-		log::info!("Received Packet {:?}", packet);
+		log::info!("Received Packet {:?}", packet.sequence);
```

Comment:
Let's remove the code for responding with a pong.

Reply:
Okay